config.m4

```diff
@@ -283,7 +283,7 @@ if test "$PHP_MONGODB" != "no"; then
   if test "$PHP_MONGODB_SYSTEM_LIBS" = "no"; then
     PHP_MONGODB_BUNDLED_CFLAGS="$STD_CFLAGS -DBSON_COMPILATION -DMONGOC_COMPILATION"
     dnl TODO: MONGOCRYPT-219 makes the -std argument obsolete
-    PHP_MONGODB_LIBMONGOCRYPT_CFLAGS="$PHP_MONGODB_BUNDLED_CFLAGS -DKMS_MSG_STATIC -std=gnu99"
+    PHP_MONGODB_LIBMONGOCRYPT_CFLAGS="-DKMS_MSG_STATIC -std=gnu99"
```

Comment:
I was going to ask why we couldn't just move this within the later conditional:

https://github.com/mongodb/mongo-php-driver/blob/1d33e149235df2b4899b9ec2d9ca5a5060f166d2/config.m4#L406-L407

However, I realized it's also set in [`libmongocrypt/CheckSSL.m4`](https://github.com/mongodb/mongo-php-driver/blob/master/scripts/autotools/libmongocrypt/CheckSSL.m4), which is included well before that point. Looking at the comment before the libmongocrypt includes, I take it this is necessary because those includes may change the value of `PHP_MONGODB_CLIENT_SIDE_ENCRYPTION` used for the conditional itself.

Reply:
That is correct. Both libmongoc/CheckSSL.m4 and libmongocrypt/CheckSSL.m4 modify their variables, which is why I decided to only inject `PHP_MONGODB_BUNDLED_CFLAGS` just before we use it.